v3/src/components/graph/components/background.tsx

```diff
@@ -56,19 +56,20 @@ export const Background = forwardRef<SVGGElement, IProps>((props, ref) => {
     height = useRef(0),
     selectionTree = useRef<typeof RTree | null>(null),
     previousMarqueeRect = useRef<rTreeRect>(),
-    currentlySelectedCaseIDs = useRef<string[]>([]),
 
-    onDragStart = useCallback((event: MouseEvent) => {
+    onDragStart = useCallback((event:any) => {
```

Comment:
Seems like there must be a better way of typing this.

Reply:
Yes, there was.